### PR TITLE
ISPN-9066 Successfully configured Slave node with no server configured is not shown in the list of hosts

### PR DESCRIPTION
**Pull request by vblagoje**
Please review @ryanemerson

### CODE REVIEW

File: src/module/server-group/ServerGroupCtrl.ts [hunk 1]
```diff
@@ -228,10 +230,10 @@ export class ServerGroupCtrl {
     return this.statuses;
   }
 
-  private filterUniqueHosts(): string[] {
-    return this.serverGroup.members
-      .map((server) => server.host)
-      .filter((item, post, array) => array.indexOf(item) === post);
+  private fetchHosts(): string[] {
+    this.domainService.getHosts().then((hosts) => {
```
Comment (ryanemerson) [inline src/module/server-group/ServerGroupCtrl.ts]: We should probably resolve this in `ServerGroup.ts` and pass to the controller.

File: src/services/domain/DomainService.ts [hunk 2]
```diff
@@ -38,6 +38,22 @@ export class DomainService {
...
+      address: [],
+      "child-type": "host"
+    };
+    let hosts: string [] = [];
+    this.dmrService.readChildResources(request).then((response) => {
+      for (let hostname in response) {
+        hosts.push(hostname);
+      }
+      deferred.resolve(hosts);
+    });
```
Comment (ryanemerson) [inline src/services/domain/DomainService.ts]: I think we can make use of `Object.keys` here and turn this into a one-liner, something like:

```typescript
this.dmrService.readChildResources(request).then(response => deferred.resolve(Object.keys(response)));
```

**Comment by vblagoje** [comment]
Updated, thanks @ryanemerson

**Comment by vblagoje** [comment]
Hey @ryanemerson is this one gtg, anything else we need to do?

**Comment by ryanemerson** [comment]
@vblagoje LGTM, can you just squash the commits and re-push please.

**Comment by vblagoje** [comment]
@ryanemerson np, squashed and forced pushed

**Pull request merged**